script.js

```diff
@@ -0,0 +1,430 @@
+  button.innerText = 'See Project';
+
+  div.appendChild(button);
+}
+// fim do loop
+
+// -------------------
+// create Details pop up
+// -------------------
+
+// // fill modal
+function fillModal(i) {
+  // create full screen container
+  const divContModal = document.createElement('div');
+  divContModal.classList = 'modal-container';
+  recentWorks.appendChild(divContModal);
+
+  // create modal
+  const divModal = document.createElement('div');
+  divModal.classList = 'details';
+  divContModal.appendChild(divModal);
+
+  // create x button
+  const xBtn = document.createElement('div');
+  xBtn.innerHTML = '<i id="xBtn" class="fa-solid fa-xmark"></i>';
+  divModal.appendChild(xBtn);
+
+  // create title of card
+  const h2 = document.createElement('h3');
+  h2.classList = 'details-title';
+  h2.innerHTML = portifolio[i].heading;
+  divModal.appendChild(h2);
+
+  // create div for language Badges
+  const divBadges = document.createElement('div');
+  divBadges.classList = 'lang';
+  divModal.appendChild(divBadges);
+
+  // create ul
+  ul = document.createElement('ul');
+  divBadges.appendChild(ul);
+
+  // create each badge items
+
+  li = document.createElement('li');
+  li.classList = 'lang-icons';
+  ul.appendChild(li);
+  div = document.createElement('div');
+  div.classList = 'badge';
+  div.innerHTML = portifolio[i].badge1;
+  li.appendChild(div);
+
+  li = document.createElement('li');
+  li.classList = 'lang-icons';
+  ul.appendChild(li);
+  div = document.createElement('div');
+  div.classList = 'badge';
+  div.innerHTML = portifolio[i].badge2;
+  li.appendChild(div);
+
+  li = document.createElement('li');
+  li.classList = 'lang-icons';
+  ul.appendChild(li);
+  div = document.createElement('div');
+  div.classList = 'badge';
+  div.innerHTML = portifolio[i].badge3;
+  li.appendChild(div);
+
+  // create img of modal
+  img = document.createElement('img');
+  img.src = (portifolio[i].image);
+  divModal.appendChild(img);
+
+  // create p
+  p = document.createElement('p');
+  p.classList = 'card-p';
+  p.innerText = portifolio[i].highlightDetails;
+  divModal.appendChild(p);
+
+  // create div to buttons
+  const divBtns = document.createElement('div');
+  divBtns.classList = 'details-btns-container';
+  divModal.appendChild(divBtns);
+
+  // create buttons
+  // see-live button
+  let anchor = document.createElement('a');
+  anchor.id = `see-live${1}`;
+  anchor.href = portifolio[i].seeLive;
+  anchor.target = '_blank';
+  divBtns.appendChild(anchor);
+
+  button = document.createElement('button');
+  button.classList = 'small-button see-project';
+  button.innerHTML = 'See Live <i class="fa-solid fa-arrow-up-right-from-square fa-flip-horizontal"></i>';
+  anchor.appendChild(button);
+
+  // see source button
+  anchor = document.createElement('a');
+  anchor.id = `source${1}`;
+  anchor.href = portifolio[i].seeSrc;
+  anchor.target = '_blank';
+  divBtns.appendChild(anchor);
+
+  button = document.createElement('button');
```

Comment:
- [ ] please make sure that you button match the ones in the Figma design and try to remove the  line under texts
- Yours 
![image](https://user-images.githubusercontent.com/61790587/175618111-c8c0cc22-1d1f-4dbd-9944-d080cbc2fcaf.png)
- Figma
![image](https://user-images.githubusercontent.com/61790587/175618138-213a136c-5b40-41ee-8dac-fb703b28479d.png)